src/afl-fuzz-run.c

```diff
@@ -169,20 +169,16 @@ write_to_testcase(afl_state_t *afl, void **mem, u32 len, u32 fix) {
 
     }
 
-    if (unlikely(afl->custom_mutators_count)) {
```

Comment:
but here you remove the if ()

Reply:
> but here you remove the if ()

https://github.com/Xeonacid/AFLplusplus/blob/f3d8bfc130b8ab36b22a456efa66e0212d0cd091/src/afl-fuzz-run.c#L98

The outter `if` is still here!
Yeah, owe have two same `if` nested originally!